Description: Fix FTBFS due to ld --no-add-needed
 This patch explicitly links the ICE libraries during build time
 to fix FTBFS caused due to --no-add-needed being passed on by
 default to LD.
Author: Bilal Akhtar <bilalakhtar@ubuntu.com>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=618108
--- a/src/audacious/Makefile
+++ b/src/audacious/Makefile
@@ -130,7 +130,7 @@
 		${REGEX_LIBS}					\
 		${LIBGUESS_LIBS}
 
-LDFLAGS += ${PROG_IMPLIB_LDFLAGS} ${AUDLDFLAGS}
+LDFLAGS += ${PROG_IMPLIB_LDFLAGS} ${AUDLDFLAGS} -lICE
 
 DBUS_BINDINGS_SOURCES = objects.xml			\
 			mpris_root.xml			\
